win32ss/user/user32/windows/cursoricon.c

```diff
@@ -2025,13 +2025,17 @@ HANDLE WINAPI CopyImage(
 {
     TRACE("hImage=%p, uType=%u, cxDesired=%d, cyDesired=%d, fuFlags=%x\n",
         hImage, uType, cxDesired, cyDesired, fuFlags);
+    HANDLE handle;
     switch(uType)
     {
         case IMAGE_BITMAP:
             return BITMAP_CopyImage(hImage, cxDesired, cyDesired, fuFlags);
         case IMAGE_CURSOR:
         case IMAGE_ICON:
-            return CURSORICON_CopyImage(hImage, uType == IMAGE_ICON, cxDesired, cyDesired, fuFlags);
+            handle = CURSORICON_CopyImage(hImage, uType == IMAGE_ICON, cxDesired, cyDesired, fuFlags);
+            if(handle == NULL && (fuFlags & (LR_COPYFROMRESOURCE|LR_COPYRETURNORG)))
```

Comment:
When you run the VB-based app in ReactOS, which flag(s) amongst these two are being specified? Both? or only one of these?

Reply:
In addition I would suggest you add here in the code a `// HACK` or `// HACKFIX` comment explaining what's happening, with the CORE-xxxx number mentioned too.

I also wonder whether this hack could be added instead into the `CURSORICON_CopyImage` function instead? In any case I guess running the test bots on that would be necessary.

Reply:
Both are being specified according to the ticket info. LR_COPYFROMRESOURCE and LB_COPYRETURNORG.

Not sure about the second comment. 
I do know that under Wine, it always show up the icons, but not sure how they do. And not 100% sure how Windows does at all. So if anyone can bring light, better than better.
I think I can't execute the test bot by myself, I have no rights for that I guess.